src/features/hotspots/root/HotspotsScreen.tsx

```diff
@@ -37,6 +39,20 @@ const HotspotsScreen = () => {
     }
   }, [])
 
+  // Set segment identity
+  const { identify } = useAnalytics()
+  const [address, setAddress] = useState<string>()
+
+  useMount(() => {
+    getAddress().then(setAddress)
+  })
+
+  useEffect(() => {
+    if (address) {
+      identify(address)
```

Comment:
Will this not set the identity again every time they open the app

Reply:
yes, this will set the identity every time and it's the expected behavior.

Reply:
The segment client is initialized in the every app launch, so it's needed to set the identity on the segment client every time.